config/postgres/sample_auctionmark_config.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<parameters>
+
+    <!-- Connection details -->
+    <type>POSTGRES</type>
+    <driver>org.postgresql.Driver</driver>
+    <url>jdbc:postgresql://localhost:5432/oltpbench?sslmode=disable&amp;ApplicationName=auctionmark&amp;reWriteBatchedInserts=true</url>
```

Comment:
Note that similarly with the mysql driver, this PR would introduce `rewriteBatchedStatements`, `reWriteBatchedInserts` by default

Reply:
yes